src/System.Collections/tests/BitArray/BitArray_CtorTests.cs

```diff
@@ -31,9 +31,8 @@ public static void Ctor_Int(int length)
                 Assert.False(bitArray[i]);
                 Assert.False(bitArray.Get(i));
             }
-            ICollection collection = bitArray;
-            Assert.Equal(length, collection.Count);
-            Assert.False(collection.IsSynchronized);
+            Assert.Equal(length, bitArray.Count);
+            Assert.False(bitArray.IsSynchronized);
```

Comment:
>Assert [](start = 12, length = 6)

also add a line to test that bitArray.SyncRoot is not null, and if you get it twice, it's the same thing each time. no need to repeat below